commander/rollup.go

```diff
@@ -115,7 +130,12 @@ func (c *Commander) unsafeRollupLoopIteration(ctx context.Context, currentBatchT
 
 	logNewBatch(batch, *commitmentsCount, duration)
 
-	err = rollupCtx.Commit()
+	err = func() error {
+		_, span := rollupTracer.Start(spanCtx, "rollupCtx.commit")
```

Comment:
Couldn't we move this one inside commit ?

Reply:
It absolutely could! Looking at it though, moving it into the commit is going to be a fair amount of change, and the next PR (the one which reworks the mempool) should get rid of the call to `Commit()`, so we'll drop this span. Because of that I think I want to leave it.